sdcm/utils/csrangehistogram.py

```diff
@@ -122,7 +122,7 @@ def _generate_percentile_name(percentile: int):
 
 _HistorgramSummary = make_dataclass("HistorgramSummary",
                                     [(_generate_percentile_name(perc), float)
-                                     for perc in PERCENTILES] + [("throughput", float)],
+                                     for perc in PERCENTILES] + [("throughput", int)],
```

Comment:
why switching to int ?

Reply:
in argus end it would be saved as float, don't know if it  make a difference

Reply:
If throughput is float it will be presented as "123.0". We do not need it float. I think.

Reply:
o.k. let move it to int, it's mostly cosmetics